### DISCUSSION TITLE
Gerador de Monstros

### DISCUSSION THREAD

**Discussion by JhanAragao** [YuriAlessandro/gerador-ficha-tormenta20#330] [Ideias]
Poderia ser de grande ajudar criar um gerador de Monstros.

**Comment by YuriAlessandro** [comment]
Como seria isso?

**Reply by Bilhalva1** [reply]
poderia ter uma lista com as possiveis características de monstros pra cada nível e ir selecionando tinha visto que o 5e tools faz isso pra D&D mas seria um trampo juntar tudo

**Comment by YuriAlessandro** [comment]
Comecei a desenvolver isso aqui hoje, mas ainda vai levar um tempo. Mas é provavelmente minha prioridade no momento.
